sdk/core/azure-core-http-okhttp/src/test/java/com/azure/core/http/okhttp/OkHttpAsyncHttpClientHttpClientTests.java

```diff
@@ -55,7 +56,8 @@ protected String getServerUri(boolean secure) {
 
     @Override
     protected HttpClient createHttpClient() {
-        return new OkHttpAsyncClientProvider().createInstance();
+        return new OkHttpAsyncHttpClientBuilder().dispatcher(createQuietDispatcher(UnexpectedLengthException.class, ""))
```

Comment:
It would be good to add a comment on why `createQuietDispatcher` is needed. Also, should we keep the log messages as this is what the customers would see anyway and we can add troubleshooting guide on what to do when this happens?

Reply:
I'll add a few comments on what this is doing, I don't think we need to have the tests spitting out uncaught error logs but I can add a bit in troubleshooting to talk about this.